frontend/packages/dev-console/integration-tests/tests/topology.scenario.ts

```diff
@@ -116,7 +116,7 @@ describe('Topology', () => {
     // Verify that there are zero connectors displayed in the Topology view
     await expect(topologyConnectors.count()).toBe(0);
 
-    // Find the source app to be connected
+    // Find the source app to be connected  
```

Comment:
please undo this change

Reply:
Net zero changes if at all possible. New lines, extra spaces, etc... That way it's easier to trace the original commit message / PR if needed.

Reply:
next time, surely I will make a note of it, but now I have multiple commits in this PR. So it would be difficult to undo this change..